packages/sanity/src/_internal/cli/server/renderDocument.ts

```diff
@@ -185,7 +185,7 @@ function getDocumentHtml(studioRootPath: string, props?: DocumentProps): string
       // If the URL is absolute, we don't need to prefix it
       return new URL(url).toString()
```

Comment:
❓ does this also need to account for basepath? I am thinking absolute url is `https://some-site.com/style.css`

Reply:
I don't think it needs to account for it - URLs like `/foo/bar` and `foo/bar` will both throw, so this is primarily for full HTTP(S) imports